apigenerator/APIGenerator.java

```diff
@@ -122,11 +140,40 @@ public void collectDependencies(LinkedList<JarFile> files, LinkedList<String> ar
 				int nClasses = m_DependencyChain.size();
 				collectDependencies(clazz);
 				System.err.format("[%d][%d]\t%s\n", m_DependencyChain.size(), m_DependencyChain.size() - nClasses, cppClassName);
+				foundClass = true;
 				break;
 			}
+			if (!foundClass)
+			{
+				if (!useSystemClasses)
+				{
+					System.err.format("\nCan't find %s class!\n\n", arg);
+					continue;
+				}
+				try
+				{
+					Class c = ClassLoader.getSystemClassLoader().loadClass(convertToBinaryClassName(arg));
+					String cppClassName = getClassName(c);
+					int nClasses = m_DependencyChain.size();
+					collectDependencies(c);
+					System.err.format("[%d][%d]\t%s\n", m_DependencyChain.size(), m_DependencyChain.size() - nClasses, cppClassName);
```

Comment:
I see this is consistent with existing code, but why are we outputting to stderr here, when it looks more like an informational log message?

Reply:
I don't know why it's done like this. I've switched non-error message to be printed to System.out in https://github.com/Unity-Technologies/android-jni-bridge/pull/41/commits/ff40b49e4ecef4d81045459090bd51f10888a735 One immediate change that I've noticed is that in the log first we see all of the system.out messages and then we get system.err messages, where previously everything was "in order". But I think it's fine to have errors listed at the end, after the normal output.